fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalValuesOperator.java

```diff
@@ -70,7 +70,8 @@ public ColumnRefSet getOutputColumns(ExpressionContext expressionContext) {
 
     @Override
     public RowOutputInfo deriveRowOutputInfo(List<OptExpression> inputs) {
-        return new RowOutputInfo(columnRefSet.stream().collect(Collectors.toMap(Function.identity(), Function.identity())));
+        return new RowOutputInfo(columnRefSet.stream().distinct()
```

Comment:
`Stream::distinct()` needs the input is sort.
> For ordered streams, the selection of distinct elements is stable (for duplicated elements, the element appearing first in the encounter order is preserved.) For unordered streams, no stability guarantees are made.

Reply:
Oh, I'm wrong, it just unstable, but the result is correct, if the input is unordered..